### ISSUE TITLE
CloudFlare a=DIUP no longer works

### ISSUE THREAD

**Issue by hdhoang** [ericpaulbishop/gargoyle#346]
The newest API is https://api.cloudflare.com/#dns-records-for-a-zone-update-dns-record which authenticates with custom headers. I'll try to figure out a command similar to the one for afraid.org.

**Comment by hdhoang** [comment]
ewget doesn't support setting headers, while wget doesn't support https, so I have to use the older api_json endpoint instead. It requires an extra field for the record ID, plus a field for the TLD of the hostname:

```
https://www.cloudflare.com/api_json.html?
tkn=[[PASSWORD]]&
email=[[EMAIL]]&
a=rec_edit&ttl=1&type=A
z=[[TLD of the DOMAIN]]&
id=[[new field here]]&
name=[[DOMAIN]]&content=[[IP]]
```

**Comment by ericpaulbishop** [comment]
Fixed (by you -- Thanks! ) in ab84e48
